### PR TITLE
Registration + Profile Activity

### PR DESCRIPTION
**Pull request by AntoineDuvalCodes**
General: Creation of the Registration Activity, user can be created from this activity to Firebase Database. + Creation of the Profile Activity, user informations can be fetched from Firebase Database and displayed inside this Activity.
Fix: Nothing
Refactor: Change of User parameter (firstName and lastName became Name)

Antoine ft Nuno

Co-Authored-By: nunopeixoto <nunopeixoto@users.noreply.github.com>

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,29 +1,45 @@
...
+        </activity>
+
+
+        <activity android:name=".RegistrationActivity"
+            android:theme="@style/AppThemeNoActionBar">
+
         </activity>
 
     </application>
-    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```
Comment (MasterMedo) [inline app/src/main/AndroidManifest.xml]: why

File: app/src/main/java/pl/put/sds/train_mate/User.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 public class User {
 
-    public String userID, firstName, lastName, pictureURL, gender;
+    public String userID, name, pictureURL, gender, email;
```
Comment (MasterMedo) [inline app/src/main/java/pl/put/sds/train_mate/User.java]: no more first names qq, we have to separate upon every fetch, or is`name an object?

File: app/src/main/java/pl/put/sds/train_mate/UserProfileActivity.java [hunk 3]
```diff
@@ -4,8 +4,13 @@
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.util.Log;
-import android.widget.TextView;
```
Comment (MasterMedo) [inline app/src/main/java/pl/put/sds/train_mate/UserProfileActivity.java]: @AntoineDuvalCodes 
rwetret

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (AntoineDuvalCodes) [reply app/src/main/AndroidManifest.xml]: I just moved it above the activities (line 5)

**Comment by MasterMedo** [comment]
I can't do anything without a google-services.json file. No way to test your work. Forgot to write it earlier.

**Pull request merged**